app/utils/validators/forms/FieldValidator.scala

```diff
@@ -136,4 +136,9 @@ object FieldValidator {
 
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
+
+  val REGEX = "^(GB|gb)[\\d]+$"
+  private val eoriPattern = Pattern.compile(REGEX)
+  val isValidEORI: String => Boolean = (name: String) => eoriPattern.matcher(name).matches()
```

Comment:
Name for this is misleading. Just reading the name suggests that it validates everything. I'd suggest maybe `isEoriStartingWithGB` ?
Other way would be to change the regex to check the digits as well.

Reply:
It is checking for digits too. The pattern ensures GB+Digits only. No characters are allowed after GB.

Reply:
isValidEORIPattern is the new name